src/main/java/com/capitalone/dashboard/collector/CollectorTask.java

```diff
@@ -62,13 +62,14 @@ public final void run() {
             LOGGER.info("Starting Collector={}", collectorName);
             long start = System.currentTimeMillis();
             collect(collector);
+            long count = collector.getLastExecutionRecordCount();
```

Comment:
You don’t have to get and set count back to collector.  It is already available

Reply:
removed unnecessary setter call